timesketch/api/v1/resources.py

```diff
@@ -1658,6 +1660,85 @@ def post(self):
             status_code=HTTP_STATUS_CODE_BAD_REQUEST)
 
 
+class AnalyzerRunResource(ResourceMixin, Resource):
+    """Resource to get all timelines for sketch."""
+
+    @login_required
+    def get(self, sketch_id):
+        """Handles GET request to the resource.
+
+        Returns:
+            A list of all available analyzer names.
+        """
+        _ = sketch_id
```

Comment:
You want to do ```sketch = Sketch.query.get_with_acl(sketch_id)``` here as well to ensure that the caller has the correct permissions

Reply:
ack